commands/unarchive.py

```diff
@@ -22,4 +37,7 @@ def unarchive(problemName):
 def unarchiveParser(parsers: ArgumentParser):
     helpText = 'Move problem from archive folder for active development.'
     parser = parsers.add_parser('unarchive', description=helpText, help=helpText)
-    parser.add_argument('problem', help='Name of problem to unarchive', nargs='+', type=problemList)
+    parser.add_argument('problem', help='Name of problem to unarchive', nargs='*', type=problemList)
+    parser.add_argument('-a', '--all', help='This flag denotes to unarchive all problems in the specified folder.', action='store_true')
+    parser.add_argument('-s', '--solved', help='This flag denotes to archive into the .solved folder', action='store_true')
```

Comment:
Remember to change the string to `unarchive` here and below
(Though I know I'm not one to talk lol)